lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -82,34 +83,31 @@ netifaces==0.10.9
 networkx==1.11
 nodeenv==1.3.3
 nose==1.3.7
-numpy==1.16.1
+numpy==1.16.2
 oauthlib==3.0.1
 openstacksdk==0.17.0
 os-client-config==1.31.2
-os-service-types==1.5.0
-osc-lib==1.12.0
-oslo.config==6.8.0
-oslo.context==2.22.0
-oslo.i18n==3.23.0
-oslo.log==3.42.2
-oslo.serialization==2.28.1
-oslo.utils==3.40.2
+os-service-types==1.6.0
+osc-lib==1.12.1
+oslo.config==6.8.1
+oslo.context==2.22.1
+oslo.i18n==3.23.1
+oslo.log==3.42.3
+oslo.serialization==2.28.2
+oslo.utils==3.40.3
 packaging==19.0
 paramiko==2.4.2
 parsley==1.3
-paste==3.0.6
+paste==3.0.7
 pastedeploy==2.0.1
 pastescript==3.0.0
-pbr==5.1.2
+pbr==5.1.3
 prettytable==0.7.2
 psutil==5.5.1
 pulsar-galaxy-lib==0.8.3
 pyasn1==0.4.5
 pycparser==2.19
 pycryptodome==3.7.3
-pycryptodomex==3.7.3
-pyinotify==0.9.6 ; sys_platform != 'win32' and sys_platform != 'darwin' and sys_platform != 'sunos5'
```

Comment:
This line should **not** be removed (pyinotify is still needed on Linux). But that's not what causing the build failure, I'm not sure what's going on there.

Reply:
I guess that was removed by the update-dependencies command; not sure what caused it though.

Reply:
I guess it's because you run `make update-dependencies` under macOS, where it's not needed.

Unfortunately, due to the fact that historically the list of Python dependencies installed by `pip` can be modified in `setup.py`, `pipenv` has to install them all to generate the `requirements.txt` files. So these files may change under different operating system or Python versions.

Reply:
So you recommend not running `make update-dependencies` under macOS?

Reply:
I'd not go that far, but deletions have to be checked.

Reply:
Sounds good, thank you for the clarification.